### PR TITLE
feat: allowed to set retryBackOff parameter of FailJobCommand

### LINKED ISSUE #481: Allow to set RetryBackOff parameter for FailJobCommand
Java and Go clients now allow to set RetryBackOff parameter: https://github.com/camunda/zeebe/issues/5629

We need the similar functionality for C# too.

### PR DESCRIPTION
**Pull request by aanodin**
closes #481

### CODE REVIEW

File: Client/Impl/Commands/FailJobCommand.cs [hunk 1]
```diff
@@ -53,6 +53,12 @@ public IFailJobCommandStep2 ErrorMessage(string errorMsg)
             return this;
         }
 
+        public IFailJobCommandStep2 RetryBackOff(TimeSpan retryBackOff)
+        {
+            request.RetryBackOff = (long)Math.Round(retryBackOff.TotalMilliseconds, MidpointRounding.AwayFromZero);
```
Comment (Zelldon) [inline Client/Impl/Commands/FailJobCommand.cs:58]: :wrench: Is this really necessary? I would feel better reducing the complexity here, and be more consistent with the other places were we use the timespan.

**Review by Zelldon — COMMENTED** [review]
Thanks @aanodin for the PR, I have one thing I would like to change, please take a look.

File: Client/Impl/Commands/FailJobCommand.cs (hunk 1 above)
Comment (aanodin) [reply Client/Impl/Commands/FailJobCommand.cs]: @Zelldon ok, I've fixed this

**Review by Zelldon — APPROVED** [review]
Thanks @aanodin for your contribution :rocket: :bow:

**Pull request merged**